components/header/src/overrides/inner.js

```diff
@@ -0,0 +1,76 @@
+/** @jsx jsx */
+
+import { jsx, useMediaQuery, getLabel } from '@westpac/core';
+
+// ==============================
+// Component
+// ==============================
+
+const Inner = ({ state: _, ...rest }) => <div {...rest} />;
+
+// ==============================
+// Styles
+// ==============================
+
+const innerStyles = (_, { fixed, scrolled }) => {
+	const mq = useMediaQuery();
+
+	return mq({
+		label: getLabel('header-inner'),
+		position: 'relative',
+		flex: 1,
+		backgroundColor: '#fff',
+		textAlign: 'left',
+		marginLeft: 'auto',
+		marginRight: 'auto',
+		marginBottom: '1px',
```

Comment:
You left this to compensate for space taken by pseudoelement which has been recently removed and replaced by a simple border-bottom.
```suggestion
```